academic_research.md

```diff
@@ -0,0 +1,33 @@
+# Chapter on outcome from DPKI academic research
+
+
+### Introduction:
+As time goes by, the use of distributed PKI takes more and more importance in regard to the one of centralized PKI. This change can be (partially/mainly) attributed to the increasing importance of IT security. On the one hand, centralization, on the other, the exact opposite. Thanks to decentralization there would be the possibility of eliminating, or reducing, the impact of the single point of failure, the CA. Which is not impenetrable to cyberattacks as  demonstrated by the 2011 [DigiNotar attack](https://www.theguardian.com/technology/2011/sep/05/diginotar-certificate-hack-cyberwar).
+The CA can be seen as a point of failure because in the worst scenario, it can be compromised and a malicious actor could create a fake certificate and trick the user into believing that it is a real one.
+
+------------
+
+Decentralized PKI can be divided into two main categories:
+- **Log-based DPKI**: This solution is mainly used to monitor the CA. Indeed certificates are not valid until they are logged on append-only public logs. Anyone can read the logs and so monitor the CA for suspect certificates. The first log-based solution is [Certificate Transparency](https://datatracker.ietf.org/doc/html/rfc6962.html), proposed by Google.
```

Comment:
Use bibliography references in this case too, by referring the RFC document by its number (as RFC-6962 in this case)